game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -99,16 +99,34 @@ function onStart(complete: () => void) {
                 tg.seq([
                     tg.immediate(context => context[miranaName].SetAttackCapability(UnitAttackCapability.NO_ATTACK)),
                     tg.moveUnit(context => context[miranaName], GetGroundPosition(Vector(600, -3700), undefined)),
-                    tg.moveUnit(context => context[miranaName], GetGroundPosition(Vector(2200, -3500), undefined)),
+                    tg.moveUnit(context => context[miranaName], GetGroundPosition(Vector(1600, -3280), undefined)),
+                    tg.immediate(context => {
+                        const mirana: CDOTA_BaseNPC_Hero = context[miranaName];
+                        setUnitVisibilityThroughFogOfWar(mirana, true);
+                        // Miiiiiiiirana: TODO maybe get a better voice line here?
+                        EmitSoundOn("Script_1_Movement_9_1", mirana);
```

Comment:
`LocalizationKey.Script_1_Movement_9_1` unless you intentionally didn't use that

Reply:
That was intentional because I did not want to signal we want to use that specific dialog line